ledger/src/blockstore.rs

```diff
@@ -3177,6 +3178,24 @@ impl Blockstore {
         self.db.storage_size()
     }
 
+    /// Returns the total physical storage size contributed by all data shreds.
+    ///
+    /// Note that the reported size does not include those recently inserted
+    /// shreds that are still in memory.
```

Comment:
> Open DB
Push in 100 shreds (assume memtable large enough to hold 100 shreds)
Close DB
Reopen DB and call this function

I need to check how it was implemented as I didn't remember all the details.  Let me double-check its current implementation and get back to you.  If I remember it correctly, it might depend on how the flush and the consistency level are set.

If we want to obtain the size of all mem-tables of one column family, we can pass "rocksdb.cur-size-all-mem-tables" to it.

Reply:
I did a little experiment here. I used `ledger-tool copy` to copy some shreds from one ledger to another. Immediately after the copy, these were the contents of the destination ledger rocksdb folder:
```
$ ls -hal ~/test/rocksdb/
total 27M
drwxrwxr-x 2 user user 4.0K Jan 13 21:16 .
drwxrwxr-x 3 user user 4.0K Jan 13 21:16 ..
-rw-r--r-- 1 user user  26M Jan 13 21:16 000005.log
-rw-r--r-- 1 user user   16 Jan 13 21:16 CURRENT
-rw-r--r-- 1 user user   37 Jan 13 21:16 IDENTITY
-rw-r--r-- 1 user user    0 Jan 13 21:16 LOCK
-rw-r--r-- 1 user user 189K Jan 13 21:16 LOG
-rw-r--r-- 1 user user 1.4K Jan 13 21:16 MANIFEST-000004
-rw-r--r-- 1 user user  74K Jan 13 21:16 OPTIONS-000075
-rw-r--r-- 1 user user  74K Jan 13 21:16 OPTIONS-000077
```
Notice no SST files; I then opened the ledger to spit out some stats with `ledger-tool analyze-storage`; here are the contents of the same folder afterwards:
```
$ ls ~/test/rocksdb/
total 27M
drwxrwxr-x 2 user user 4.0K Jan 13 21:27 .
drwxrwxr-x 3 user user 4.0K Jan 13 21:16 ..
-rw-r--r-- 1 user user  51K Jan 13 21:27 000079.sst
-rw-r--r-- 1 user user  938 Jan 13 21:27 000080.sst
-rw-r--r-- 1 user user 235K Jan 13 21:27 000081.sst
-rw-r--r-- 1 user user  26M Jan 13 21:27 000082.sst
-rw-r--r-- 1 user user 1.1K Jan 13 21:27 000083.sst
-rw-r--r-- 1 user user 1.2K Jan 13 21:27 000084.sst
-rw-r--r-- 1 user user  984 Jan 13 21:27 000085.sst
-rw-r--r-- 1 user user    0 Jan 13 21:27 000087.log
-rw-r--r-- 1 user user   16 Jan 13 21:27 CURRENT
-rw-r--r-- 1 user user   37 Jan 13 21:16 IDENTITY
-rw-r--r-- 1 user user    0 Jan 13 21:16 LOCK
-rw-r--r-- 1 user user 178K Jan 13 21:27 LOG
-rw-r--r-- 1 user user 189K Jan 13 21:16 LOG.old.1642109274093204
-rw-r--r-- 1 user user 2.5K Jan 13 21:27 MANIFEST-000086
-rw-r--r-- 1 user user  74K Jan 13 21:16 OPTIONS-000077
-rw-r--r-- 1 user user  73K Jan 13 21:27 OPTIONS-000089
```
So. it appears that opening the database caused the write-ahead-log to be persisted into SST's